nodes/persistence-node/package.json

```diff
@@ -10,7 +10,8 @@
     "/bin"
   ],
   "bin": {
-    "wrap-persistence-node": "./bin/main.js"
```

Comment:
After looking at go-ipfs, I think we should use a single file for the commands
The app name here will be "wrap-persistence-node"
The command that needs to be added is "daemon"
You can rename the "API" command to "daemon"

So example usage will look like "wrap-persistence-node daemon -l --http 8080" 
This will spin up both the IPFS gateway API and the persistence node API (port 6051)